File: official/docs/golang/current/endshipper/create.go
package example

import (
	"fmt"
	"os"

	"github.com/EasyPost/easypost-go/v2"
)

func main() {
	apiKey := os.Getenv("EASYPOST_API_KEY")
	client := easypost.New(apiKey)

	endshipper, _ := client.CreateEndShipper(
		&easypost.Address{
			Name:    "FOO BAR",
			Company: "BAZ",
			Street1: "164 TOWNSEND STREET UNIT 1",
			Street2: "UNIT 1",
			City:    "SAN FRANCISCO",
			State:   "CA",
			Zip:     "94107",
			Country: "US",
			Phone:   "555-555-5555",
			Email:   "FOO@EXAMPLE.COM",
		},
	)

	fmt.Println(endshipper)
}

File: official/docs/ruby/current/carbon-offset/create.rb
require 'easypost'

EasyPost.api_key = ENV['EASYPOST_API_KEY']

EasyPost::Shipment.create(
  {
    to_address: {
      name: 'Dr. Steve Brule',
      street1: '179 N Harbor Dr',
      city: 'Redondo Beach',
      state: 'CA',
      zip: '90277',
      country: 'US',
      phone: '4155559999',
      email: 'dr_steve_brule@gmail.com',
    },
    from_address: {
      name: 'EasyPost',
      street1: '417 Montgomery Street',
      street2: '5th Floor',
      city: 'San Francisco',
      state: 'CA',
      zip: '94104',
      country: 'US',
      phone: '4153334445',
      email: 'support@easypost.com',
    },
    parcel: {
      length: 20.2,
      width: 10.9,
      height: 5,
      weight: 65.9,
    },
  },
  nil,
  true,
)

puts shipment
